src/pages/resultsView/coExpression/CoExpressionViz.tsx

```diff
@@ -284,17 +294,31 @@ export default class CoExpressionViz extends React.Component<ICoExpressionVizPro
 
     @bind
     private table() {
-        return (
-            <div>
-                <CoExpressionTable
-                    referenceGene={this.props.gene}
-                    dataStore={this.dataStore}
-                    tableMode={this.dataStore.tableMode}
-                    onSelectTableMode={this.onSelectTableMode}
-                />
-                {this.requestAllDataButton}
-            </div>
-        );
+        if (this.props.profileY.molecularAlterationType !== 'GENESET_SCORE'){
```

Comment:
AlterationTypeConstants.GENESET_SCORE - lets always use constants rather than "magic strings", because you can't catch magic string typos with typescript